Makefile.am

```diff
@@ -813,6 +813,18 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	$(srcdir)/installation/install_clear_image.sh $(CLEAR_VM_IMAGE_VERSION) $(IMAGE_PATH)
```

Comment:
oh ok, so you said two steps to instal the kernel and the image? one for just downloading other for installing (last one will be run as root)